nodes/scheduling/nodes-custom-scheduler.adoc

```diff
@@ -0,0 +1,26 @@
+[id="nodes-custom-scheduler"]
+= Running a custom scheduler
+include::modules/common-attributes.adoc[]
+:context: nodes-custom-scheduler
+
+toc::[]
+
+You can run multiple custom schedulers alongside the default scheduler and configure which scheduler to use for each pod.
+
+[IMPORTANT]
+====
+It is supported to use a custom scheduler with {product-title}, but Red Hat does not directly support the functionality of the custom scheduler.
```

Comment:
@sferich888 @reestr Can you please review this statement?

Reply:
+1 

However we may want to point users at https://docs.openshift.com/container-platform/4.7/nodes/scheduling/nodes-scheduler-default.html and https://docs.openshift.com/container-platform/4.7/nodes/scheduling/nodes-scheduler-default.html#nodes-scheduler-default-modifying_nodes-scheduler-default to explain how and in what ways they can configure the OCP scheduler in a supported way.

Reply:
Good suggestion! Updating to include a link